src/OmikronFactFinder.php

```diff
@@ -5,14 +5,95 @@
 namespace Omikron\FactFinder\Shopware6;
 
 use Omikron\FactFinder\Shopware6\Export\Field\FieldInterface;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepositoryInterface;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\EqualsAnyFilter;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\IdSearchResult;
 use Shopware\Core\Framework\Plugin;
+use Shopware\Core\Framework\Plugin\Context\InstallContext;
+use Shopware\Core\Framework\Plugin\Context\UninstallContext;
+use Shopware\Core\System\CustomField\CustomFieldTypes;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
 class OmikronFactFinder extends Plugin
 {
+    private const CUSTOM_FIELD_NAME = 'cms_export_selector';
+
     public function build(ContainerBuilder $container): void
     {
         parent::build($container);
         $container->registerForAutoconfiguration(FieldInterface::class)->addTag('factfinder.export.field');
     }
+
+    //TODO: Should we add custom field on install or on plugin activation (activate method)
```

Comment:
IMHO it doesn't really matter.
but i would remove the field set and the field on `uninstall`
Did you try to install the module, uninstall and install again? Didn't repository throw an exception saying that set already exist?

Reply:
Added a check if user data was kept on last uninstall before creating custom field